### DISCUSSION TITLE
I subscribed to the video downloader yesterday and I want a refund, who can I talk to?

### DISCUSSION THREAD

**Discussion by kick852** [aclap-dev/video-downloadhelper#1176] [Help and Q&A]
### Your question

I subscribed to the video downloader yesterday and I want a refund, who can I talk to?

### Addon & CoApp version and details

version: 9.1.0.43
target: google
channel: stable
lang: pt-BR
coapp: {"found":true,"new_version":false,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19"}
license: {"accepted":true,"email":"<removed>","key":"46140a8fe65a41cd"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
{
    "valid_license_message_has_been_displayed": true,
    "never_show_no_incognito_msg_again": true,
    "view_options": {
        "all_tabs": false,
        "hide_downloaded": false,
        "low_quality": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "sort_by_status": true,
        "sort_reverse": false
    },
    "use_sidebar": true,
    "open_count_store": 20,
    "successfull_dl": 58,
    "dable_attempts": {
        "__serializer_tag": "map",
        "__serializer_value": [
            {
                "__serializer_tag": "array",
                "__serializer_value": [
                    {
                        "__serializer_tag": "primitive",
                        "__serializer_value": "acesso.ohardcopy.com.br"
                    },
                    {
                        "__serializer_tag": "array",
                        "__serializer_value": [
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 71
                            },
                            {
                                "__serializer_tag": "primitive",
                                "__serializer_value": 58
                            }
                        ]
                    }
                ]
            }
        ]
    },
    "dable_attempts_last": 1724276433921
}

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
